dataland-frontend/src/components/resources/companyCockpit/ClaimOwnershipPanel.vue

```diff
@@ -1,10 +1,8 @@
 <template>
   <div class="claim-panel">
     <div class="next-to-each-other vertical-middle">
-      <h2 class="claim-panel__ownership-question" data-test="claimOwnershipPanelHeading">
```

Comment:
![image](https://github.com/d-fine/Dataland/assets/104447806/3e4466a4-0ebe-46a9-9828-e497e64f782c)
![image](https://github.com/d-fine/Dataland/assets/104447806/2a69c7a7-2015-46f1-848c-b2686e0036c9)
Wenn ich die Implementierung (1. Bild) mit dem Figma (2.Bild) vergleiche, frage ich mich gerade ob man die Verifiziert Markierung noch einen Stück tiefer schieben sollte und ob die Ecken noch runder sein sollten. Was meinst du?

Reply:
sind dran, das anzupassen. Die Höhe passt in meinen Augen, es ist jeweils mittig ausgerichtet an den Großbuchstaben. Die Rundung geht einfach.

Reply:
angepasst

Reply:
ok